rust/sbd-server/src/cslot.rs

```diff
@@ -0,0 +1,416 @@
+//! Attempt to pre-allocate as much as possible, including our tokio tasks.
+//! Ideally this would include a frame buffer that we could fill on ws
+//! recv and use ase a reference for ws send, but alas, fastwebsockets
+//! doesn't seem up to the task. tungstenite will willy-nilly allocate
+//! buffers for us, but at least we should only be dealing with one at a
+//! time per connection.
+
+use super::*;
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex, Weak};
+
+static U: std::sync::atomic::AtomicU64 = std::sync::atomic::AtomicU64::new(1);
```

Comment:
The unique id is only used at runtime by the server to distinguish multiple client connections that may have incorrectly re-connected using the same pub key. It is not needed to persist this across restarts, because the new server will not have any connections to start.

TCP connections cannot survive a server restart without doing really weird stuff like passing file handles off to other processes, that I have no intention of implementing : )